game/quake2_md2.ksy

```diff
@@ -0,0 +1,429 @@
+  |  160-168 | crattak | 1-9    |                                                        |
+  |  169-172 |  crpain | 1-4    |                                                        |
+  |  173-177 | crdeath | 1-5    |                                                        |
+  |  178-183 |  death1 | 01-06  |                                                        |
+  |  184-189 |  death2 | 01-06  |                                                        |
+  |  190-197 |  death3 | 01-08  |                                                        |
+  ```
+
+  The above are filled in for player models; for the separate weapon models,
+  the final frame is 173 "g_view" (unknown purpose) since weapons aren't shown
+  during death animations. `a_grenades.md2`, the handgrenade weapon model, is
+  the same except that the `wave` frames are blank (according to the default
+  female model files). This is likely due to its dual use as a grenade throw
+  animation where this model must leave the player's model.
+doc-ref:
+  - https://icculus.org/~phaethon/q3a/formats/md2-schoenblum.html
+  - http://tfc.duke.free.fr/coding/md2-specs-en.html
+  - http://tastyspleen.net/~panjoo/downloads/quake2_model_frames.html
+  - http://wiki.polycount.com/wiki/OldSiteResourcesQuake2FramesList
+seq:
+  - id: magic
+    contents: IDP2
+  - id: version
+    type: u4
+    valid: 8
+  - id: skin_width_px
+    type: u4
+  - id: skin_height_px
+    type: u4
+  - id: bytes_per_frame
+    type: u4
+  - id: num_skins
+    type: u4
+  - id: vertices_per_frame
+    type: u4
+  - id: num_tex_coords
+    type: u4
+  - id: num_triangles
+    type: u4
+  - id: num_gl_cmds
+    type: u4
+  - id: num_frames
+    type: u4
+  - id: ofs_skins
+    type: u4
+  - id: ofs_tex_coords
+    type: u4
+  - id: ofs_triangles
+    type: u4
+  - id: ofs_frames
+    type: u4
+  - id: ofs_gl_cmds
+    type: u4
+  - id: ofs_eof
+    type: u4
+instances:
+  skins:
+    pos: ofs_skins
+    size: 64
+    type: strz
+    encoding: ascii
+    repeat: expr
+    repeat-expr: num_skins
+  tex_coords:
+    pos: ofs_tex_coords
+    type: tex_point
+    repeat: expr
+    repeat-expr: num_tex_coords
+  triangles:
+    pos: ofs_triangles
+    type: triangle
+    repeat: expr
+    repeat-expr: num_triangles
+  frames:
+    pos: ofs_frames
+    type: frame
+    repeat: expr
+    repeat-expr: num_frames
+  gl_cmds:
+    pos: ofs_gl_cmds
+    # in http://tfc.duke.free.fr/coding/src/md2.c initially read as `int *`
+    # (a pool of `int`s) => hence `s4`
+    size: 4 * num_gl_cmds # FIXME: replace `4` with `sizeof<s4>` once `sizeof` is implemented for GraphViz
+    type: gl_cmds_list
+  anim_names:
+    value: |
+      ['stand', 'run', 'attack', 'pain1', 'pain2', 'pain3', 'jump', 'flip',
+      'salute', 'taunt', 'wave', 'point', 'crstnd', 'crwalk', 'crattak',
+      'crpain', 'crdeath', 'death1', 'death2', 'death3']
+  anim_start_indexes:
```

Comment:
```suggestion
  anim_start_indices:
```

Please don't forget to change the occurrence in `/doc` too:

https://github.com/kaitai-io/kaitai_struct_formats/blob/2f1b418810ff416aaab92efef57d9a6328d4f2d0/game/quake2_md2.ksy#L41-L42